src/content-handlers/iiif/extensions/uv-default-extension/config/config.json

```diff
@@ -134,7 +134,8 @@
         "fullScreen": "$fullScreen",
         "moreInfo": "$moreInfo",
         "open": "$open",
-        "share": "$share"
+        "share": "$share",
+        "toggleLeftPanel": "Toggle Left Panel"
```

Comment:
All of these other instances should be converted to use $toggleLeftPanel like the uv-av-extension.